Tests/LibWeb/Text/expected/CSSOMView/mouse-event-x-y-values.txt

```diff
@@ -0,0 +1,10 @@
+50
+50
+50
+50
+50
+50
+50
+60
+42
+52
```

Comment:
could we also print the names of the event properties? it's a bit hard to tell if they're correct by looking at a list of numbers.

Reply:
Done.